tuftool/src/remove_key_role.rs

```diff
@@ -55,44 +53,16 @@ pub(crate) struct RemoveKeyArgs {
 
 impl RemoveKeyArgs {
     pub(crate) fn run(&self, role: &str) -> Result<()> {
-        // load the repo
-        // We don't do anything with targets so we will use metadata url
-        let settings = tough::Settings {
-            root: File::open(&self.root).unwrap(),
-            datastore: None,
-            metadata_base_url: self.metadata_base_url.to_string(),
-            targets_base_url: self.metadata_base_url.to_string(),
-            limits: Limits::default(),
-            expiration_enforcement: ExpirationEnforcement::Safe,
-        };
-
-        // Load the `Repository` into the `TargetsEditor`
-        // Loading a `Repository` with different `Transport`s results in
-        // different types. This is why we can't assign the `Repository`
-        // to a variable with the if statement.
-        if self.metadata_base_url.scheme() == "file" {
-            let repository = Repository::load(Box::new(FilesystemTransport), settings)
-                .context(error::RepoLoad)?;
-            self.with_targets_editor(
-                role,
-                TargetsEditor::from_repo(repository, role)
-                    .context(error::EditorFromRepo { path: &self.root })?,
-            )?;
-        } else {
-            let repository = Repository::load(Box::new(HttpTransport::new()), settings)
-                .context(error::RepoLoad)?;
-            self.with_targets_editor(
-                role,
-                TargetsEditor::from_repo(repository, role)
-                    .context(error::EditorFromRepo { path: &self.root })?,
-            )?;
-        }
-
-        Ok(())
+        let repository = load_metadata_repo(&self.root, &self.metadata_base_url)?;
+        self.remove_key_with_targets_editor(
+            role,
+            TargetsEditor::from_repo(repository, role)
+                .context(error::EditorFromRepo { path: &self.root })?,
+        )
     }
 
-    /// Removes keys from adelegated role using targets Editor
-    fn with_targets_editor(&self, role: &str, mut editor: TargetsEditor) -> Result<()> {
+    /// Removes keys from a delegated role using targets Editor
+    fn remove_key_with_targets_editor(&self, role: &str, mut editor: TargetsEditor) -> Result<()> {
```

Comment:
nit: rename similar methods in other modules to fit a pattern?  For example, I see a similar change in remove_role.rs, but not add_key_role.rs.

Reply:
Addressed in [59899dd](https://github.com/awslabs/tough/compare/3e858e5b5d629576a0c13f3f7b627cd8c49af9a4..59899ddf735ceb7e4edc883a9b1c6a195506f820),